src/components/shared/ExerciseSide.tsx

```diff
@@ -0,0 +1,7 @@
+import '../../styles/ExerciseSide.scss';
+
+function ExerciseSide(): JSX.Element {
+  return <section id="exercise-side-container"></section>;
```

Comment:
```suggestion
  return <section id="exercise-side-container">
  <LevelSelector/>
  </section>;
```

and create a `LevelSelector.ts` file that has `return <div>level selector</div>` or something